test/quantization/test_quant_api.py

```diff
@@ -637,6 +637,22 @@ def test_quantized_model_to_device(self):
         cuda_res = m(*example_inputs_cuda)
         self.assertEqual(cuda_res.cpu(), ref)
 
+    # TODO: enable this test
```

Comment:
why disable test?

Reply:
cpu -> cuda does not work yet, I changed it to cuda to cuda for now